sympy/solvers/tests/test_inequalities.py

```diff
@@ -486,3 +495,178 @@ def test__pt():
     assert _pt(x, oo) == _pt(oo, x) == x + 1
     assert _pt(x, -oo) == _pt(-oo, x) == x - 1
     raises(ValueError, lambda: _pt(Dummy('i', infinite=True), S.One))
+
+
+def test_linprog_from_matrices():
+    # There's no need to have many test cases because test_linprog_from_equations
+    # and test_find_feasible test the simplex function extensively.
+
+    # test matricies with wrong shapes
+    A = Matrix([[1,0], [0, 1]])
+    B = Matrix([[2],[4]])
+    C = Matrix([[3, 1, 3]])
+    raises(ValueError, lambda: linprog_from_matrices(A, B, C))
+    B = Matrix([[2],[4], [5]])
+    C = Matrix([[3, 1]])
+    raises(ValueError, lambda: linprog_from_matrices(A, B, C))
+    B = Matrix([[2],[4], [5]])
+    C = Matrix([[3, 1, 3]])
+    raises(ValueError, lambda: linprog_from_matrices(A, B, C))
+
+def test_linprog_maximize_from_equations():
```

Comment:
```suggestion

def test_linprog_maximize_from_equations():
```

Reply:
If the output of `linprog_from_matrices` is the same as `linprog_maximize_from_equations` can we drop `maximize` from the function name to just have `linprog_from_eqs`?